docs/book/src/developers/development.md

```diff
@@ -553,7 +553,7 @@ With the following environment variables defined, you can build a CAPZ cluster f
 | ------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `E2E_ARGS`                | `-kubetest.use-ci-artifacts`                                                                                                                                                                             |
 | `KUBERNETES_VERSION`      | `latest` - extract Kubernetes version from https://dl.k8s.io/ci/latest.txt (main's HEAD)<br>`latest-1.<MINOR>` - extract Kubernetes version from dl.k8s.io/ci/latest-1.<MINOR>.txt (release branch's HEAD) |
-| `WINDOWS_SERVER_VERSION`  | Optional, can be `windows-2019` (default) or `windows-2022`                                                                                                                                              |
+| `WINDOWS_SERVER_VERSION`  | Optional, can be `windows-2022` (default) or `windows-2025`                                                                                                                                              |
```

Comment:
Are we done building images for 2019? Bumping the default seems reasonable then. Let's keep an eye on jobs that rely on this implicit default.